Makefile

```diff
@@ -8,17 +8,20 @@ SERVICE_MODEL_NAME=$(shell echo $(MODEL_NAME))
 ifeq ($(SERVICE_MODEL_NAME),)
   SERVICE_MODEL_NAME=""
 endif
+AWS_SDK_GO_VERSION=$(shell echo $(SDK_VERSION))
+ACK_RUNTIME_VERSION=$(shell echo $(RUNTIME_VERSION))
+TEST_INFRA_COMMIT_SHA=$(shell echo $(COMMIT_SHA))
```

Comment:
I guess users will have to provide `SDK_VERSION`, `RUNTIME_VERSION` and `COMMIT_SHA` before running the make rule.

Reply:
I do not think this will work James,

if users do not set value of $SDK_VERSION, an empty string will be assigned to AWS_SDK_GO_VERSION and the default (curl command) will not get assigned.

Try to follow this answer to see if it works:
https://stackoverflow.com/a/54009169